src/Core/Framework/DataAbstractionLayer/Indexing/EntityIndexerRegistry.php

```diff
@@ -17,10 +16,6 @@
 
 /**
  * @final
- *
- * @internal
```

Comment:
the real solution here would be to split the class in the internal message handler and a public registry right?